README.md

```diff
@@ -429,6 +429,7 @@ Current number of checkpoints: ![](https://img.shields.io/endpoint?url=https://h
 1. **[Pix2Struct](https://huggingface.co/docs/transformers/model_doc/pix2struct)** (from Google) released with the paper [Pix2Struct: Screenshot Parsing as Pretraining for Visual Language Understanding](https://arxiv.org/abs/2210.03347) by Kenton Lee, Mandar Joshi, Iulia Turc, Hexiang Hu, Fangyu Liu, Julian Eisenschlos, Urvashi Khandelwal, Peter Shaw, Ming-Wei Chang, Kristina Toutanova.
 1. **[PLBart](https://huggingface.co/docs/transformers/model_doc/plbart)** (from UCLA NLP) released with the paper [Unified Pre-training for Program Understanding and Generation](https://arxiv.org/abs/2103.06333) by Wasi Uddin Ahmad, Saikat Chakraborty, Baishakhi Ray, Kai-Wei Chang.
 1. **[PoolFormer](https://huggingface.co/docs/transformers/model_doc/poolformer)** (from Sea AI Labs) released with the paper [MetaFormer is Actually What You Need for Vision](https://arxiv.org/abs/2111.11418) by Yu, Weihao and Luo, Mi and Zhou, Pan and Si, Chenyang and Zhou, Yichen and Wang, Xinchao and Feng, Jiashi and Yan, Shuicheng.
+1. **[Pop2Piano](https://huggingface.co/docs/transformers/main/model_doc/pop2piano)** released with the paper [Pop2Piano : Pop Audio-based Piano Cover Generation](https://arxiv.org/abs/2211.00895) by Jongho Choi and Kyogu Lee.
```

Comment:
@amyeroberts As the first author of Pop2Piano and the developer of the model, may I make a comment that might be helpful here?

Pop2Piano was started when I was a graduate student at Seoul National University, and I completed the research and submitted this paper after I graduated.

Therefore, if an institution must be mentioned in this doc, the most appropriate institution name would be "Seoul National University". If it is based on the time when the model was published, it would be correct not to mention the institution because I was an individual researcher at that time.

(The company I work for, Rebellions Inc. listed in the paper, is an NPU company, and I did not research Pop2Piano as part of my work for this company, so it does not seem appropriate to list it).

Reply:
As @sweetcocoa has said - "it would be correct not to mention the institution because I was an individual researcher at that time." lets not add any company or research institution here and I am marking this as Resolved.